process-engine/bpmn-studio
# ♻️🐛 Do not Generate a New Camunda Property Objects

## Changes

1. Fix the new generation of a `<camunda:Properties>` element, if a ServiceTask already has an empty PropertiesElement.
2. Remove empty ExtensionElements and PropertieElements when last Property gets deleted.
3. Fix generating an empty Property when initializing a ServiceTask.
4. Refactor ExternalTask, HTTP-Service-Task and ServiceTask component and add ServiceTaskService.

## Issues

Closes #1525 
Closes #1553 

PR: #1534

## How can others test the changes?

- Add a ServiceTask
- Configure a HTTP ServiceTask and run the process
- be sure that all works as expected
- repeat these steps with an ExternalTask
- Check that the XML does not contain empty Properties/PropertiesElement or ExtensionElements, only if a HTTP ServiceTask or ExternalTask is configured.
